components/centraldashboard/public/components/landing-page.js

```diff
@@ -67,41 +66,8 @@ export class LandingPage extends mixinBehaviors([AppLocalizeBehavior], utilities
         ns = ns
             .replace(/[^\w]|\./g, '-')
             .replace(/^-+|-+$|_/g, '')
-            .replace(/[0-9]/g, '')
             .toLowerCase();
-
-        this.getNamespaces(ns);
-    }
-
-    async getNamespaces(ns) {
-        await fetch(
-            `/api/namespaces/`, {
-                method: 'GET',
-                headers: {
-                    'Content-Type': 'application/json',
-                },
-            }
-        )
-            .then((res) => res.json())
-            .then((data) => {
-                const namespaceNames = [];
-                data.forEach((element) => {
-                    namespaceNames.push(element.metadata.name);
-                });
-                let counter = 1;
-                // Remove any numbers at the end
-                const originalNs = ns.replace(/\d+/g, '');
-                if (namespaceNames.includes(originalNs)) {
-                    while (namespaceNames.includes(originalNs + counter)) {
-                        counter++;
-                    }
-                    ns = originalNs + counter;
-                }
-                this.namespaceName = ns;
-            }).catch((e)=> {
-                this.showError('landingPage.errGeneral');
```

Comment:
I don't think the "errGeneral" message, as well as the "errorDetail" prop are used anywhere else. If so, those should be cleaned up.